docs/README.md

```diff
@@ -404,6 +405,39 @@ Scan the directories were the Dockerfiles are to feed the scanner.
       - uses: Alfresco/alfresco-build-tools/.github/actions/docker-scan-image-dirs@ref
 ```
 
+### enforce-pr-conventions
+
+Checks if the branch name and pull request title follow conventions.
+
+```yaml
+      - uses: Alfresco/alfresco-build-tools/.github/actions/enforce-pr-conventions@ref
+        with:
+          jira-project-key: AAE|HXCS # optional
+          valid-branch-regex: "^(feature|test|tmp)\/AAE-[0-9]+-[A-Za-z0-9._-]+$" # optional
+          valid-pr-title-regex: "^AAE-[0-9]+ .+$" # optional
```

Comment:
As is, this sample is not consistent as not all inputs are needed at once, and only one project is referenced in the regex shown as samples whil 2 projects are shown in key.

I think it would make sense to show 2 use cases:
- first use case (most simple), you only define jira-project-key (and here we explain that multiple projects can be used with "|" as separator
- second use case (advanced), you only define regex inputs: the project key is ignored anyway, in this case you can define your regular expressions as you wish (including project key(s))

Reply:
Done.